src/muParserDLL.cpp

```diff
@@ -192,7 +192,7 @@ API_EXPORT(const muChar_t*) mupGetVersion(muParserHandle_t a_hParser)
         muParser_t* const p(AsParser(a_hParser));
 
 #ifndef _UNICODE
-    sprintf(s_tmpOutBuf, "%s", p->GetVersion().c_str());
+    snprintf(s_tmpOutBuf, 2048, "%s", p->GetVersion().c_str());
 #else
     wsprintf(s_tmpOutBuf, _T("%s"), p->GetVersion().c_str());

```

Comment:
Seems there's also a wnsprintf function for unicode.

Reply:
Yes, I could not find an equivalent either and left it alone.

Reply:
But it does exist.

Reply:
Sorry about that, I misread your comment, I will add wnsprintf() for unicode. I won't be able to test it though.